drivers/modem/simcom-y7080.c

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (C) 2023 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT simcom
```

Comment:
driver is not compatible for all the simcom, pls restrict only for 7080 and y7080. e.g drivers/sensor/tmp108/tmp108.c